iChef-WebUi/src/app/components/user-profile-page/user-profile-page.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component } from '@angular/core';
+import { PATHS } from 'src/app/constants/paths';
+import { ACCOUNT_MENU_LINKS } from 'src/app/constants/account-page';
+
+@Component({
+  selector: 'app-user-profile-page',
+  templateUrl: './user-profile-page.component.html',
+  styleUrls: ['./user-profile-page.component.scss']
+})
+export class UserProfilePageComponent {
+
```

Comment:
We should have a userId input here, and based on that make a request to get the user with the given id to retrieve information about the user